src/community/propose-a-component-or-pattern/index.md.njk

```diff
@@ -15,7 +15,7 @@ Follow the steps to propose a component or pattern for the Design System.
 
 ## 1. Check the list of discussions on GitHub
 
-Check if someone else has already suggested your idea or something similar on the [list of discussions on GitHub](https://github.com/orgs/alphagov/projects/43/views/2).
+Check if someone else has already suggested your idea or something similar on the [Community Backlog project board on GitHub](https://github.com/alphagov/govuk-design-system-backlog/projects/1).
```

Comment:
Alright @owenatgov – basically this is the only content that looks wrong to me. It should be the same as what's on main i.e. "list of discussions" etc. Tried to add a commit to change it back but getting lost in more awful conflict resolution. You ok to take this change out? Then we're good to go 👍